app/src/androidTest/java/co/nimblehq/util/ViewActions.kt

```diff
@@ -0,0 +1,27 @@
+package co.nimblehq.util
+
+import android.view.View
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers
+import org.hamcrest.Matcher
+
+
+/**
+ * Created by Viktor Artemiev on 2019-08-07.
+ * Copyright (c) 2019, Nimble. All rights reserved.
+ */
+
+fun waitFor(delay: Long) = object : ViewAction {
+        override fun getConstraints(): Matcher<View> {
```

Comment:
trivial, but we should format the file properly all the time for the correct indentation 😇

Reply:
done